treemap/app/index.html

```diff
@@ -87,6 +87,11 @@ <h1 class="treemap-placeholder__heading">Lighthouse Treemap</h1>
   <script src="https://www.gstatic.com/firebasejs/4.1.2/firebase-auth.js"></script>
   <script src="src/bundled.js"></script>
 
+  <script>
+    // b/298230436
+    window.__hash = window.location.hash;
+    window.location.hash = '';
+  </script>
```

Comment:
Should this be above `bundled.js` and the other scripts?

Reply:
It looks at both, so either way 🤷